### PR TITLE
Update ASM to 7.0 for full Java 11 support

### PR DESCRIPTION
**Pull request by ijuma**
* Removed dependency on `org.ow2.asm:asm-xml` which has been removed.
* Set ASM version to ASM7 unconditionally.
----

Make sure these boxes are checked before submitting your PR -- thank you!

- [x] Added an entry into `CHANGELOG.md` if you have changed SpotBugs code

### CODE REVIEW

**Comment by wreulicke** [comment]
asm-xml is deleted from 7.0-beta.
https://asm.ow2.io/versions.html

**Comment by ijuma** [comment]
Thanks @wreulicke, fixed it.

**Comment by KengoTODA** [comment]
I remember that we use asm-xml to keep backward compatibility in Eclipse manifest, see #400 and 
 e7c8920e8db3e687639ea0f690c3f3a5333575ad for detail.

@iloveeclipse could you review this PR? I'm not sure we can remove `asm-xml` safely or not.

**Comment by iloveeclipse** [comment]
We can remove xml package, we have not re-exported it, but we should remove it completely :-)

**Comment by ijuma** [comment]
Thanks, updated the PR.

File: spotbugs/META-INF/MANIFEST-TEMPLATE.MF [hunk 1]
```diff
@@ -65,7 +65,6 @@ Export-Package: edu.umd.cs.findbugs,
  org.objectweb.asm.tree,
  org.objectweb.asm.tree.analysis,
  org.objectweb.asm.util,
```
Comment (iloveeclipse) [inline spotbugs/META-INF/MANIFEST-TEMPLATE.MF]:
```suggestion
 org.objectweb.asm.util
```

**Pull request merged**